### ISSUE TITLE
Fragpipe analyst error

### ISSUE THREAD

**Issue by tygrad** [Nesvilab/FragPipe#1170]
- Describe the issue or question:
You closed the two requests. Can you please provide an answer.
I uploaded the combined protein_tsv and experiment annotation on fragpipe analyst but it says error. At the second attempt, it says disconnected from server and I tried to reload but couldnot connect to the server.

- Upload your log file
(If a log file hasn't been generated, go to the 'Run' tab in FragPipe, click 'Export Log', zip the resulting "log_[date_time].txt" file to avoid truncation, then attach the zipped file by drag & drop here.)

**Comment by fcyu** [comment]
> You closed the two requests. Can you please provide an answer.

Sorry for the confusion. I did not closed both requests. I just closed this one (https://github.com/Nesvilab/FragPipe/issues/1169) because you created a duplicated one here (https://github.com/Nesvilab/FragPipe-Analyst/issues/14) which is still open. And the second one is more appropriate to stay open because your question if about FragPipe-Analyst: "_After analysis with fragpipe, I'm unabale to analyse my data for differential expression using Fragpipe analyst_". (FragPipe finished without any issues but you had problems with FragPipe-Analyst.)

BTW, I will also close this one because it is another duplicated one. You probability need to wait for reply in https://github.com/Nesvilab/FragPipe-Analyst/issues/14

Best,

Fengchao
